.travis.yml

```diff
@@ -3,3 +3,7 @@ node_js:
   - "0.12"
   - "0.11"
   - "iojs"
+
+before_script:
+    - npm install -g jshint jscs
+    - npm install
```

Comment:
Please add `jshint` and `jscs` to `devDependencies` in package.json and revert back the changes in `.travis.yml`.

That's the only right way how to ensure that `npm install && npm test` works on all machines.